github-review.el

```diff
@@ -633,7 +682,7 @@ Github API provides only the originalPosition in the query.")
           (github-review-save-diff
            (a-assoc pr-alist 'sha .data.repository.pullRequest.headRef.target.oid)
            (github-review-format-diff (-first-item x) .data.repository.pullRequest)
-	   (a-assoc pr-alist 'sha .data.repository.pullRequest.baseRef.target.oid)))))
+	   (a-assoc pr-alist 'sha .data.repository.pullRequest)))))
```

Comment:
A solitary commit review aka 3.0. Commit `bc..` should reflect that in api call results.

Reply:
5.1 on `bc..` (to expect original-commit-id as that).